src/main/java/life/qbic/portal/portlet/connection/ConnectionHandler.java

```diff
@@ -84,7 +84,7 @@ public void setCredentials() {
     openBisUser = conf.getDataSourceUser();
     LOG.info("openBIS user = {}", openBisUser);
     openBisPw = conf.getDataSourcePassword();
-    openBisUrl = conf.getDataSourceApiUrl();
+    openBisUrl = conf.getDataSourceUrl();
```

Comment:
Is there a reason why this data source url does not already have the `/openbis/openbis` suffix?

Reply:
Maybe the ApiUrl is a different configuration property?

Reply:
Might be, this method was deprecated therefore I removed it. Is this ok with you?

Reply:
```suggestion
    openBisUrl = conf.getDataSourceUrl() + "/openbis/openbis" ;
```

Reply:
why not keeping it where it is?

Reply:
because it is part of the actual URL you use later on. So as a developer, I would expect the `openBisUrl` to not require any modification when I want to use it in the code somewhere.

Reply:
Right, but the URL would still not be valid since there are other (OpenBis specific) components that need to be added. And this assembly is all at the same place. So I would prefer to leave it as is.

Reply:
Sorry, @jenniferboedker I do not understand what you are saying here. What other open bis components?

Reply:
Up to you